### ISSUE TITLE
OnClick console error on every page

### ISSUE THREAD

**Issue by issuereporter1** [markjaquith/page-links-to#101]
After updating to 3.3.1 I am running into the following JS error.  It is not a blocker for my site at this time, however JS errors like this can cause strange behavior so I want to report it.

/wp-content/plugins/page-links-to/dist/new-tab.js?ver=3.3.1

```
new-tab.js?ver=3.3.1:1 Uncaught TypeError: Cannot read property 'tagName' of null
    at n (new-tab.js?ver=3.3.1:1)
    at HTMLDocument.o (new-tab.js?ver=3.3.1:1)
n @ new-tab.js?ver=3.3.1:1
o @ new-tab.js?ver=3.3.1:1
new-tab.js?ver=3.3.1:1 Uncaught TypeError: Cannot read property 'tagName' of null
    at n (new-tab.js?ver=3.3.1:1)
    at HTMLDocument.o (new-tab.js?ver=3.3.1:1)
n @ new-tab.js?ver=3.3.1:1
o @ new-tab.js?ver=3.3.1:1
new-tab.js?ver=3.3.1:1 Uncaught TypeError: Cannot read property 'tagName' of null
    at n (new-tab.js?ver=3.3.1:1)
    at HTMLDocument.o (new-tab.js?ver=3.3.1:1)
n @ new-tab.js?ver=3.3.1:1
o @ new-tab.js?ver=3.3.1:1
new-tab.js?ver=3.3.1:1 Uncaught TypeError: Cannot read property 'tagName' of null
    at n (new-tab.js?ver=3.3.1:1)
    at HTMLDocument.o (new-tab.js?ver=3.3.1:1)
n @ new-tab.js?ver=3.3.1:1
o @ new-tab.js?ver=3.3.1:1
```

**Comment by demaier** [comment]
Same behavior seen here.

https://wordpress.org/support/topic/latest-version-plugin-conflict/

**Comment by markjaquith** [comment]
Thanks all! I just pushed v3.3.2 with a fix.

**Comment by ace-hellas** [comment]
After upgrading to 3.2.2 version the following error appears on click in every page

char:1992
object does not support property or method "closest"
https://www.scadapro.com/wp-content/plugins/page-links-to/dist/new-tab.js?ver=3.3.2

**Comment by markjaquith** [comment]
@ace-hellas Using Internet Explorer?

**Comment by ace-hellas** [comment]
Yes. I have pages included inside a windows software and the default browser is IE.

**Comment by markjaquith** [comment]
@ace-hellas thanks for the reply! Working on a fix for IE right now.

**Comment by markjaquith** [comment]
@ace-hellas I released version 3.3.3 which I've verified does not throw JS errors in IE 9.

IE 8 is... just not happening.

**Comment by E-VANCE** [comment]
Hey Mark,

I am seeing the error for Mobile Safari UI/WKWebView 12.1.4 as well:

`TypeError: Object doesn't support property or method 'closest'`

Traceback says: `https://christian-sander.net/wp-content/plugins/page-links-to/dist/new-tab.js line 1 col 14888 in o`

Haven't seen it for other browsers (yet), updated the plugin to 3.3.3 yesterday.

Kind regards!
